release-team/role-handbooks/release-notes/README.md

```diff
@@ -38,19 +40,29 @@ The doc will be edited by SIG leads and SIG members but will also be edited for
 grammar and uniform style by the release notes team. The changelog must also be
 generated and the release notes tool must continue to be run on the release
 branch in order to pull in any outstanding PRs that are merged between the
-beginning of code freeze and the release.
+beginning of code freeze and the release. The same applies to the website, which
+should always represent the same state as the markdown document.
 
 
 ## Tasks
 
 - The Release Notes Lead and Shadows attend burn down meetings, SIG Release meetings and follow the [#sig-release](https://kubernetes.slack.com/messages/C2C40FMNF) Slack channel for relevant information throughout the release cycle.
 - One member of the Release Notes Team should be responsible for [setting up and running](setup-tool.md) the [release-notes tool](https://github.com/kubernetes/release/tree/master/cmd/release-notes) to collect generated drafts of all release-notes identified in the current release -- improvements to the tool are always welcome!
 - Drafts produced by the tool are copied to a file called `release-notes-draft.md` in the [releases folder](../../../releases) for the current release in the `sig-release` repo.
+- The [release-notes website](https://github.com/kubernetes-sigs/release-notes)
+  should be updated together with the markdown based version: The website uses the
+  JSON output from the release-notes tool to render it online. We publish the
+  intermediate version directly to the `master` branch to reflect the current
+  state. For this, the previous release notes have to be preserved which means
+  that we have to edit the JSON manually for now. Contributions and enhancements
```

Comment:
To me this seems to be the most critical part. We would have multiple options in my point of view, where I (currently) suggest to append the new notes to the previous ones (and override them for every run). WDYT?

Reply:
We don't have to edit the JSON manually. When you run the `release-notes` tool and specify an existing (valid) release notes JSON file, it will append automatically. 

Unless I misunderstand the comment?

Reply:
Nice, I did not knew that. So then I will remove my comment regarding manually editing. :+1: